File: src/api/Models/Tenant.cs
namespace Shadowbrook.Api.Models;

public class Tenant
{
    public Guid Id { get; set; }
    public required string OrganizationName { get; set; }
    public required string ContactName { get; set; }
    public required string ContactEmail { get; set; }
    public required string ContactPhone { get; set; }
    public DateTimeOffset CreatedAt { get; set; }
    public DateTimeOffset UpdatedAt { get; set; }

    // Navigation
    public ICollection<Course> Courses { get; set; } = new List<Course>();
}

File: src/web/src/types/tenant.ts
export interface Tenant {
  id: string;
  organizationName: string;
  contactName: string;
  contactEmail: string;
  contactPhone: string;
  courseCount?: number;
  createdAt: string;
  updatedAt: string;
}

export interface TenantDetail extends Tenant {
  courses: Array<{
    id: string;
    name: string;
  }>;
}
